packages/expo-file-system/src/FileSystem.ts

```diff
@@ -111,9 +112,7 @@ export async function getContentUriAsync(fileUri: string): Promise<string> {
     }
     return await ExponentFileSystem.getContentUriAsync(fileUri);
   } else {
-    return new Promise(function (resolve, reject) {
-      resolve(fileUri);
-    });
+    return new Promise((resolve) => resolve(fileUri));
```

Comment:
You can just return it
```suggestion
    return fileUri;
```